modules/ROOT/pages/sso-prerequisites-about.adoc

```diff
@@ -35,5 +35,4 @@ https://anypoint.mulesoft.com/accounts/login/{your_org_domain}
 * OpenID Connect does not support role mapping.
 * If a SAML user belongs to certain groups, Anypoint Platform does not automatically grant equivalent roles in the organization.
```

Comment:
Just a note: I reread this and I'm not sure that I understand how "certain groups" relates to "equivalent roles in the organization." If your audience does, then this is OK.